.coderabbit.yaml

```
@@ -49,20 +51,31 @@ reviews:
         The README of `aru-prism-prison` covers this folder quite well
     - path: 'aru-processor-core/**/*.java'
       instructions: |
-        This folder contains the dense code that wires together the processor. As per `CONTRIBUTING.md`, this folder should really only have:
+        This folder contains the dense code that wires together the processor. The `module-info` specifies that the code is `@NullMarked`, therefore
+          if code is added that can put `null` somewhere that isn't explicitly annotated as `Nullable` it should be called out.
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)